src/kirby/components/list/list.component.scss

```diff
@@ -5,6 +5,13 @@ $background-color-hover: get-color('background-color');
 $shadow: get-elevation(2);
 $item-height: size('xxxl');
 
+ion-list.has-divider {
```

Comment:
Following my personal opinion, i would like this rule to be moved into the `ion-list` rule on line 34.

By doing this we run the risk having split styling rules all over the `scss` file, making it hard to read.

Reply:
Fully agree - thanks.